### DISCUSSION TITLE
Missing 'timestamps' property for Globals configuration

### DISCUSSION THREAD

**Discussion by wojciechkrol** [payloadcms/payload#642] [Feature Requests & Ideas]
Hello devs,

I see a little inaccuracy in configuration of globals in compare to collections. Every collection item has timestamps by default. For collections we can disable this feature, but for globals we can't, although timestamps are attached to response.

Can you add feature to disable timestamps for globals? Now I can do it using hooks but it would be great to disable it by property.

Thank you!

**Comment by jmikrut** [comment]
Hey @wojciechkrol — this should be totally doable. 

I'm not sure when we could commit to it, but we'd be happy to build this when we have time. We'd also accept a PR for this functionality!
